package.json

```diff
@@ -69,7 +69,7 @@
     "updates": "^6.2.1"
   },
   "engines": {
-    "node": "10.2.x"
+     "node": ">=10.2 <10.15.1"
```

Comment:
This looks confusing, what does it do?

Reply:
For regular users using npm this is just an indicator on which versions of node that can run this project fine.
If you use an alternative dependency manager, like Yarn, you'll get an error if your node version doesn't match the ones listed under "engines".
My change only extends the list of working node versions to include the versions of node between the last set version(10.2.x) and the latest recommended version listed on https://nodejs.org/en/ (which I use and it works without problems).

Not really an important change, want me to revert it?